lib/cartopy/io/img_tiles.py

```diff
@@ -421,22 +421,37 @@ def __init__(self, access_token, map_id, cache=False):
             A valid Mapbox API access token.
         map_id
             An ID for a publicly accessible map (provided by Mapbox).
-            This is the map whose tiles will be retrieved through this process.
-
+            This is the map whose tiles will be retrieved through this process
+            and is specified through the Mapbox Styles API
+            (https://docs.mapbox.com/api/maps/styles/)
+            Examples:
+              map_id='mapbox/streets-v11'
```

Comment:
Is there a reason to keep the `mapbox/` in the id, versus just making this `streets-v11`? It looks like all Mapbox Styles have the prefix `mapbox/` so you could put that into the base string.

Reply:
Sorry to be slow responding here (teaching !).  I left it this way because it matched the documentation that is provided by mapbox and so is a copy/paste from their list (and I also wondered if they were reserving the option to have other urls by keeping it this way). But whatever the community wants and is used to, that's fine with me.

Reply:
No worries at all!

Their documentation is a little bit confusing to parse, but I _think_ the `mapbox/` section is the `{username}` portion, so they are giving examples with that as the username and we should really just hardcode that on our end?

I played around with this to make an API call: https://docs.mapbox.com/playground/static/

and on page load I get:
`https://api.mapbox.com/styles/v1/{username}/{style_id}/static`
and moving around I get:
`https://api.mapbox.com/styles/v1/mapbox/streets-v11/static`